pom.xml

```diff
@@ -48,16 +48,16 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.16.0</flink.version>
+        <flink.version>1.17-SNAPSHOT</flink.version>
```

Comment:
Why do we depend on a SNAPSHOT version here? It seems to block the release of this connector, that before flink 1.17 release, no new pulsar connector can release.

cc @zentol @syhily

Reply:
> Why do we depend on a SNAPSHOT version here?

There is no Flink 1.17 released yet, that's the next to-be released version. Pulsar has added support for the watermark alignment of source splits https://issues.apache.org/jira/browse/FLINK-28853 which is only available in Flink 1.17